pallas-common/src/test/java/com/vip/pallas/utils/IPUtilsTest.java

```diff
@@ -0,0 +1,69 @@
+package com.vip.pallas.utils;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import java.net.InetSocketAddress;
+import java.net.UnknownHostException;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNull;
+
+public class IPUtilsTest {
+
+  @Rule public final ExpectedException thrown = ExpectedException.none();
+
+  @Test
+  public void testGetHostAddress() {
+    assertNull(IPUtils.getHostAddress(null));
+    assertEquals("192.168.1.1:255", IPUtils.getHostAddress(new InetSocketAddress("192.168.1.1", 255)));
+  }
+
+  @Test
+  public void testGetHostName() {
+    assertEquals("127.0.1.1", IPUtils.getHostName("127.0.0.1"));
```

Comment:
I think getHostName() should be passed a string which represents a hostname rather than an ip-address.
Also, this example looks wrong in that "127.0.0.1" != "127.0.1.1"